deploy-manage/upgrade/deployment-or-cluster.md

```diff
@@ -28,17 +28,39 @@ products:
 
 # Upgrade your deployment or cluster [upgrade-deployment-cluster]
 
-When upgrading an existing cluster, you perform a minor or major upgrade. For example, a minor upgrade takes you from version 9.0.0 to 9.1.0, while a major upgrade takes you from version 8.0.0 to 9.0.0.
+This section contains the actual upgrade instructions for {{es}} clusters and {{kib}} instances. Upgrade procedures depend on whether you installed Elastic components using Elastic-managed or self-managed infrastructure.
 
-Upgrade procedures depend on whether you installed Elastic components using Elastic-managed or self-managed infrastructure.
+## Prerequisites
+
+Before proceeding with the upgrade, review the [Plan your upgrade](/deploy-manage/upgrade/plan-upgrade.md) guidance to understand compatibility and timing considerations, and follow the steps in [Prepare to upgrade](/deploy-manage/upgrade/prepare-to-upgrade.md) to get your environment ready for the upgrade.
+
+## Out-of-order releases [out-of-order-releases]
+
+Elastic maintains several minor versions of Elasticsearch at once. This means releases do not always happen in order of their version numbers. You can only upgrade to {{version.stack}} if the version you are currently running meets both of these conditions:
+
+* Has an older version number than {{version.stack}}
```

Comment:
```suggestion
* Has a lower version number than {{version.stack}}
```
(for consistency, because up above we say "if the version number is higher")

Reply:
In this case we are saying `older` but actually `older version number`, so clearly we are identifying the numeric / ordinal value of the version.

But you are right that if we keep `higher`, probably lower makes more sense here, and `lower version number` should be fine too.

Thanks! I hadn't realized about this.